client/hotrod-client/src/test/java/org/infinispan/client/hotrod/marshall/NonIndexedEmbeddedRemoteQueryTest.java

```diff
@@ -25,10 +24,4 @@ protected ConfigurationBuilder createConfigBuilder() {
       builder.encoding().value().mediaType(MediaType.APPLICATION_OBJECT_TYPE);
       return builder;
    }
-
-   @Test(expectedExceptions = HotRodClientException.class, expectedExceptionsMessageRegExp = "org.infinispan.objectfilter.ParsingException: ISPN028521: Full-text queries cannot be applied to property 'longDescription' in type sample_bank_account.Transaction unless the property is indexed and analyzed.")
-   @Override
-   public void testRemoteFullTextQuery() {
-      super.testRemoteFullTextQuery();
-   }
```

Comment:
I removed the test since I had to change the base implementation, that is `EmbeddedRemoteInteropQueryTest#testRemoteFullTextQuery`.
The change is caused by the fact that Hibernate Search 6 does not support a field that is sortable and full text at the same time.

Anyway, I think that we can keep the original implementation for the subclass, so that we can keep the test case. I'm doing that...